infra/terraform/aks/modules/postgresql/main.tf

```diff
@@ -0,0 +1,46 @@
+resource "random_password" "db_password" {
+  length           = 16
+  special          = true
+  override_special = "_%@"
+}
+
+resource "azurerm_postgresql_server" "postgresql" {
+  name                = var.name
+  location            = var.location
+  resource_group_name = var.resource_group_name
+
+  administrator_login          = var.administrator_login
+  administrator_login_password = random_password.db_password.result
+
+  sku_name          = var.sku_name
+  version           = var.postgresql_version
+  storage_mb        = var.storage_mb
+  auto_grow_enabled = false
+
+  public_network_access_enabled    = true
+  ssl_enforcement_enabled          = true
+  ssl_minimal_tls_version_enforced = "TLS1_2"
+
+  tags = {
+    environment = "ecosystem-ci"
+  }
+}
+
+resource "azurerm_postgresql_database" "databases" {
+  for_each = toset(var.databases)
+
+  name                = each.value
+  resource_group_name = var.resource_group_name
+  server_name         = azurerm_postgresql_server.postgresql.name
+  charset             = "UTF8"
+  collation           = "English_United States.1252"
+}
+
```

Comment:
Any chance you can also add `schema_migrations` table to `registry` database?

Reply:
So, besides the database we also need to create this table? According to the Harbor documentation:
`Harbor will create tables automatically when starting up.`

Reply:
Most of the tables, but not this one I assume. Or why else is it created in the initial set of SQL commands (that are executed by init container for the internal database case)? 

https://github.com/goharbor/harbor/blob/master/make/photon/db/initial-registry.sql#L4

Reply:
I believe that unless this is documented we should not add this or we might end up with CI environments different than what customers do.